Aplib.Core/Intent/Actions/QueryAction.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using Aplib.Core.Belief;
-using Aplib.Core.Intent.Actions;
+using System;
```

Comment:
Could you remove using System? It sometimes creates issues with our Action class vs the System.Action delegate.